azuretest/modules/db/module.ftl

```diff
@@ -39,6 +39,31 @@
                                     "Prefix": "MOCK"
                                 }
                             }
+                        },
+                        "mysqldb" : {
+                            "db" : {
+                                "Instances" : {
+                                    "default" : {
+                                        "DeploymentUnits" : [ "solution-az-mysqldb" ]
+                                    }
+                                },
+                                "Profiles" : {
+                                    "Testing" : [ "Component" ]
```

Comment:
The profile is no longer called `Component`

Reply:
Thanks missed that